superset/assets/javascripts/explorev2/components/SelectField.jsx

```diff
@@ -37,6 +37,19 @@ export default class SelectField extends React.Component {
       this.props.onChange(this.props.name, optionValue);
     }
   }
+  renderOption(opt) {
+    if (this.props.name === 'viz_type') {
+      const url = `/static/assets/images/viz_thumbnails/${opt.value}.png`;
+      const noImg = '/static/assets/images/noimg.png';
+      return (
+        <div>
+          <img className="viz-thumb-option" src={url} alt={noImg} />
```

Comment:
img's `alt` props is used for accessibility and as text that shows up when you hover and image. You should just put the name of the viz in there if anything
http://www.w3schools.com/tags/att_img_alt.asp